src/components/dataset/query/QueryView.jsx

```diff
@@ -125,8 +125,9 @@ export class QueryView extends React.PureComponent {
   };
 
   hasDataset() {
-    const { dataset } = this.props;
-    return dataset && dataset.schema;
+    const { dataset, match } = this.props;
+    const datasetId = match.params.uuid;
```

Comment:
will match always have a params val?

Reply:
oh, I have no idea... I would think so, since the UUID is part of the path for each dataset. It's being used just like this in componentDidMount